linux_os/guide/system/auditing/auditd_local_events/oval/shared.xml

```diff
@@ -0,0 +1,6 @@
+{{{
+oval_auditd_config(
+    parameter="local_events",
+    value="yes"
```

Comment:
Is this the default value? if so, then include missing_parameter_pass=true.

Reply:
Man page `man 5 auditd.conf` says default value is yes. But I don't know what happens if the option isn't present in config file at all.

Reply:
Based on the source code of `audit` we can infer that even if the option does not exist in the file it will still have the default value set:
https://github.com/linux-audit/audit-userspace/blob/be5c84c19171dda26f565205aec38376b6d9834a/src/auditd-config.c#L306

IMO, it is safe to set `missing_parameter_pass=true`.

Reply:
@ggbecker Great finding! I'll add it.

Reply:
@ggbecker added

Reply:
Great. Thanks. Resolving conversation.